### PR TITLE
Angular 5

### PR DESCRIPTION
**Pull request by tunnel**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,12 +1,14 @@
 {
   "name": "ng2-auto-complete",
-  "version": "0.5.4",
+  "version": "0.5.5",
```
Comment (allenhwkim) [inline package.json]: This must be 0.6 since it is a major update. It might be a breaking change. If so, please update it to 1.0.0

**Comment by allenhwkim** [comment]
@ndcunningham Please review, test, and merge this PR. It has major changes, but I can't keep up with Angular breaking changes(3/4/5) since I don't work with it.

**Comment by ndcunningham** [comment]
i don't understand why this commit is made from this branch?
complete rewrite from `webpack.config.js` with no changes to `package.json` for angular upgrade to v5 @tunnel @allenhwkim . What's going on here?

**Comment by allenhwkim** [comment]
@ndcunningham I donno what's happening here. 0.5.5? Regarding maintainer,  If you want to maintain this repo, please send me your npm id, I will give you full permission of npm publish.

**Comment by ndcunningham** [comment]
@allenhwkim you don't want to maintain this anymore?
It was regarding this PR with what i highlighted being: Rewrite of webpack and no version update for angular 5 when the title of the PR is angular 5

**Comment by allenhwkim** [comment]
@ndcunningham no, I don't want to maintain this. Since I am not using this package at my work place(not using Angular2-5), I found myself not active with this package, and decided to share/pass the ownership to someone who is active.

**Comment by almothafar** [comment]
This is outdated and does not look like it will work, closing it.

**Pull request closed without merging**